app/controllers/admins/employees_controller.rb

```diff
@@ -33,6 +33,25 @@ def update
       render :edit, notice: e.message
     end
 
+    def add_available_kudos
+      if params[:number_of_additional_kudos].to_i.between? 1, 20
```

Comment:
Just a detail, but now this controller action looks like:
```
if condition
  lots of stuff
  lots of stuff
  lots of stuff
else
  one line of code
end
```
It can be hard to read, especially when we have `rescue`.

You can change the structure to more readable format, like:
```
unless condition
  return error message
end

lots of stuff
lots of stuff
lots of stuff
```
So: enter the method and immediately return if the condition is not met. Otherwise, continue.

Reply:
Changed, it looks much better - thanks 🙂